neard/src/log_config_watcher.rs

```diff
@@ -0,0 +1,87 @@
+use actix::{Actor, Arbiter, Context};
+use near_o11y::reload_env_filter;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+use tokio::runtime::Handle;
+
+/// Runs LogConfigWatcher in a separate thread.
+pub(crate) fn spawn_log_config_watcher(watched_path: PathBuf) {
+    let log_config_arbiter = Arbiter::new();
+    let log_config_arbiter_handle = log_config_arbiter.handle();
+    LogConfigActor::start_in_arbiter(&log_config_arbiter_handle, move |_ctx| LogConfigActor {
+        watched_path,
+    });
+}
+
+/// Configures logging.
+#[derive(Serialize, Deserialize, Clone, Debug)]
+struct LogConfig {
+    /// Comma-separated list of EnvFitler directives.
+    pub rust_log: Option<String>,
+    /// Some("") enables global debug logging.
+    /// Some("module") enables debug logging for "module".
+    pub verbose_module: Option<String>,
```

Comment:
As per my comment on the other PR, I’d get rid of verbose_module. This is an ill-conceived option it’s better if we let it fall into obscurity and eventually remove `--verbose` command line option.

Reply:
`--verbose` has an important advantage. Enabling `debug` but blacklisting noisy sources of log messages such as `h2`. Without the `--verbose` flag, the user would need to care to add `h2=warn` and other directives every time.

Reply:
You can just specify `module_you_care_about=debug` in the log levels

Reply:
Even if you specify just `debug`, the rules to reverse the level for `h2` and similar can be unconditionally applied after the user-supplied directives. Though the downside in that case would be that it would no longer be possible to increase the logging for these modules.

Reply:
By the way, just to be clear, my dislike of `--verbose` is due to the flag taking a required argument.